services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java

```diff
@@ -20,47 +20,48 @@ interface DirectionsService {
-   * @param profile                       {@link RouteOptions#profile()}
-   * @param coordinates                   {@link RouteOptions#coordinatesList()}
-   * @param accessToken                   the access token
-   * @param alternatives                  {@link RouteOptions#alternatives()}
-   * @param geometries                    {@link RouteOptions#geometries()}
-   * @param overview                      {@link RouteOptions#overview()}
-   * @param radiuses                      {@link RouteOptions#radiuses()}
-   * @param steps                         {@link RouteOptions#steps()}
-   * @param bearings                      {@link RouteOptions#bearings()}
-   * @param avoidManeuverRadius           {@link RouteOptions#avoidManeuverRadius()}
-   * @param layers                        {@link RouteOptions#layers()}
-   * @param continueStraight              {@link RouteOptions#continueStraight()}
-   * @param annotations                   {@link RouteOptions#annotations()}
-   * @param language                      {@link RouteOptions#language()}
-   * @param roundaboutExits               {@link RouteOptions#roundaboutExits()}
-   * @param voiceInstructions             {@link RouteOptions#voiceInstructions()}
-   * @param bannerInstructions            {@link RouteOptions#bannerInstructions()}
-   * @param voiceUnits                    {@link RouteOptions#voiceUnits()}
-   * @param exclude                       {@link RouteOptions#exclude()}
-   * @param include                       {@link RouteOptions#include()}
-   * @param approaches                    {@link RouteOptions#approaches()}
-   * @param waypointIndices               {@link RouteOptions#waypointIndices()}
-   * @param waypointNames                 {@link RouteOptions#waypointNames()}
-   * @param waypointTargets               {@link RouteOptions#waypointTargets()}
-   * @param enableRefresh                 {@link RouteOptions#enableRefresh()}
-   * @param walkingSpeed                  {@link RouteOptions#walkingSpeed()}
-   * @param walkwayBias                   {@link RouteOptions#walkwayBias()}
-   * @param alleyBias                     {@link RouteOptions#alleyBias()}
-   * @param snappingIncludeClosures       {@link RouteOptions#snappingIncludeClosures()}
-   * @param snappingIncludeStaticClosures {@link RouteOptions#snappingIncludeStaticClosures()}
-   * @param arriveBy                      {@link RouteOptions#arriveBy()}
-   * @param departAt                      {@link RouteOptions#departAt()}
-   * @param maxHeight                     {@link RouteOptions#maxHeight()}
-   * @param maxWidth                      {@link RouteOptions#maxWidth()}
-   * @param computeTollCost               {@link RouteOptions#computeTollCost()}
-   * @param waypointsPerRoute             {@link RouteOptions#waypointsPerRoute()}
-   * @param metadata                      {@link RouteOptions#metadata()}
+   * @param userAgent                          the user agent
```

Comment:
Do we need to change the formatting? 🤔

Reply:
Oh, I added the parameter and then removed it. But the formatting stayed. I'll revert it.